Gemfile

```
@@ -0,0 +1,59 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
```

use gem mysql2 instead of sqlite3

---

So I have to delete the sqlite3 and install mysql2? How to do it?

---

remove sqlite3 and add
```suggestion
gem "mysql2"
```
and run command
bundle install

---

Done it. Thank you very much.